csv/generic/src/test/scala/fs2/data/csv/generic/AutoDerivationTest.scala

```diff
@@ -40,7 +40,6 @@ class AutoDerivationTest extends AnyFlatSpec with Matchers {
   }
 
   it should "prefer custom decoders over derived ones" in {
-    import auto._
```

Comment:
These auto imports should no be removed, they are what is under test (the fact they their are not used)